torchgeo/datasets/nasa_marine_debris.py

```diff
@@ -236,7 +236,7 @@ def plot(
         image = draw_bounding_boxes(image=sample["image"], boxes=sample["boxes"])
```

Comment:
Can you do the same thing for "boxes"? Are there any images in the original dataset without positive examples?